src/components/Table/TableHeader.vue

```diff
@@ -4,11 +4,16 @@ import { h } from 'vue';
 export default {
   name: 'TableHeader',
   render () {
-    const columns = this.$slots
-      .default()
+    const index = this.$slots
```

Comment:
Can we simplify this even more? (I actually was going to open a PR too 🤗 ) This code snippet does the same thing but will pass props through instead of stripping them off the slot items
```
    const columns = this.$slots.default().map((slotItem) => h('td', slotItem));
    const tr = h('tr', columns);
    return h('thead', tr);
```